psalm.xml.dist

```diff
@@ -75,5 +75,15 @@
                 <file name="Repository/ServiceEntityRepository.php"/>
             </errorLevel>
         </DuplicateClass>
+        <InternalClass>
```

Comment:
needed for using `Symfony\Bridge\Doctrine\Middleware\Debug\Query` in the new test case I added. Or I can remove the test again? :man_shrugging:

Reply:
Add a psalm-suppress in an offending code instead please